packages/coq/coq.9.0.0/opam

```diff
@@ -7,10 +7,7 @@ homepage: "https://coq.inria.fr/"
 doc: "https://coq.github.io/doc/"
 bug-reports: "https://github.com/coq/coq/issues"
 depends: [
-  "dune" {>= "3.8"}
-  "rocq-prover" {= version}
   "coq-core" {= version}
   "coq-stdlib" {= "9.0.0"}
```

Comment:
Probably also
```suggestion
  "coq-stdlib"
```

Reply:
At the time, the consensus was that packages that are still depending on the legacy `coq` package also expect the version of `coq-stdlib` to be strongly tied to the version of `coq-core`. I do not have a strong opinion on whether we should revisit this decision and remove the strict versioning.

Reply:
I am not sure I understand. Don't one risk to have broken switches suddenly if the stdlib is not matching the coq version?